en/concept/architecture.md

```diff
@@ -29,7 +29,7 @@ again. -->
 The source code is split into self-contained modules/programs (shown in `monospace` in the
 diagram). Usually a building block corresponds to exactly one module. 
 
-> **Tip** At runtime, you can inspect which modules are executed with the `top` command, 
+> **Tip** At runtime, you can inspect which modules are executed with the `top` command in nuttx shell, 
```

Comment:
Should it be NuttX shell or nsh shell? Either way, not nuttx

Reply:
I'd say just `shell`. While `top` is NuttX-specific, the rest can be used in the SITL shell (`pxh>`) as well.